packages/cli/src/database.ts

```diff
@@ -37,3 +45,61 @@ export const convertToIdentifiers = <T extends Table>({ table, fields }: T, with
     fields: Object.fromEntries(fieldsIdentifiers) as FieldIdentifiers<keyof T['fields']>,
   };
 };
+
+/**
+ * Note `undefined` is removed from the acceptable list,
+ * since you should NOT call this function if ignoring the field is the desired behavior.
+ * Calling this function with `null` means an explicit `null` setting in database is expected.
+ * @param key The key of value. Will treat as `timestamp` if it ends with `_at` or 'At' AND value is a number;
+ * @param value The value to convert.
+ * @returns A primitive that can be saved into database.
+ */
+// eslint-disable-next-line complexity
+export const convertToPrimitiveOrSql = (
```

Comment:
This seems a duplication of `convertToPrimitiveOrSql` in @logto/core, should we move this to some other packages (such as @logto/shared) as a global util function?

Reply:
<img width="454" alt="image" src="https://user-images.githubusercontent.com/14722250/194698125-a20d2af4-cce9-410f-a667-8f16f975cfd6.png">